src/sentry/dynamic_sampling/latest_release_booster.py

```diff
@@ -161,3 +164,47 @@ def add_boosted_release(project_id: int, release_id: int, environment: Optional[
         datetime.utcnow().replace(tzinfo=UTC).timestamp(),
     )
     redis_client.pexpire(cache_key, BOOSTED_RELEASE_TIMEOUT * 1000)
+
+
+@dataclass(frozen=True)
+class BoostedRelease:
+    version: str
+    environment: Optional[str]
+    platform: Platform
+    timestamp: float
+
```

Comment:
@dataclass
class BostedReleases:
     releases: List[BoostedRelease]
     
     def get_release_ids(self) -> int:
          return [self.release_id for release in self.releases]